src/components/CCIP/Tables/ChainTable.tsx

```diff
@@ -54,80 +54,82 @@ function ChainTable({ lanes, explorerUrl, sourceNetwork, environment }: TablePro
         />
         <TableSearchInput search={search} setSearch={setSearch} />
       </div>
-      <table className="ccip-table">
-        <thead>
-          <tr>
-            <th>{inOutbound === LaneFilter.Outbound ? "Destination" : "Source"} network</th>
-            <th>{inOutbound === LaneFilter.Outbound ? "OnRamp" : "OffRamp"} address</th>
-            <th>Status</th>
-          </tr>
-        </thead>
-        <tbody>
-          {lanes
-            ?.filter((network) => network.name.toLowerCase().includes(search.toLowerCase()))
-            .map((network, index) => (
-              <tr key={index}>
-                <td>
-                  <div
-                    className="ccip-table__network-name"
-                    role="button"
-                    onClick={() => {
-                      const laneData = getLane({
-                        sourceChain: sourceNetwork.key as SupportedChain,
-                        destinationChain: network.key as SupportedChain,
-                        environment,
-                        version: Version.V1_2_0,
-                      })
+      <div className="ccip-table__wrapper">
```

Comment:
Wrong margin should be 24
![image](https://github.com/user-attachments/assets/a6d7978d-5fda-4478-998d-af6aa836324e)